examples/gateway-tlsroute.yaml

```diff
@@ -59,8 +59,8 @@ kind: Secret
+  tls.key: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2QUlCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktZd2dnU2lBZ0VBQW9JQkFRRGV6SWdXUTgvVythTHgKVERPanZvOGY5OVpGYVBoOWQrTkFYQ1NmSTFLcS9TdVdrMnJyVzErK3ZDMDgwRFZOdzh2bHdnRVFRSWFXdzZtWAp6V2hQNmppdHIvemxSVGg4TWFwSFMvTXhXbjN0WnFKZ3ZVdVoyQWcxMFdQTXhQdXhRSVpTYW5xT3kzRE14MkNwCmlMQ1c0SVBERlRhQm5XT1hOeFg4bEMvQit6QlZYYzBIYVdUU2pQVWJRRk40ZUZwQW1weXFqTUNqTndjVVJ3cEEKVSs0cXpDeVZ2ZU5VU0RLWHpoN04rUDlPRkFiVjNqL0IyOXpqT2xUVko1NS9nZVR4YmplVkJrRkNlcC9CSHhQRgo4MnhtWUJOQnJ2WVU5dFkyc0JCZmh6OGFUNFJaMmx5NXJxVnZGdnhMMXVHdmZTb0J5R2hVNUVYODQ2ZlVibm5zCmxJSDdBKzdOQWdNQkFBRUNnZ0VBR2V4dmV6TzV0N2hGT3RGVWd0MTExdERtQ0I1K0owOHZUSHBNTW80UUhPQW4KVmJaaWJ1OHJWVXljOE9JRENjRFo5U3BkS0ZCeHVKSWNwaEthN0EyRE9jWnI1UnRvUnA4YXduLzJWSXhFY2I2Swo3em1uVmNlNnNOUmp5ZHd6c3Z5Z2hHOXFRam91cmswV2xMTllaTDNMNXdPTlc0aCtJWWFoMHNsRlEwdGdaTlkxCkNSSEZ5ZThnSHlBV2pEMWpqNitNVFFPZ2tGVTJoZmhuMThsTXh3NG9OZnp6a1dKcVlsWXFZa0ZUSGpZTnYwS3UKWWV0dkQxNWRjd05uYXhxQTF6WFlUVDNGYkpBT3h3Y2pXNlF4V3A5K3JTbzFlc2FHYWxKT2RIVWlmUmlkT1Q5YgowdndxT2gwM3phck91Z3lHdS9uQUZqcTFpRVN4dVBuV2EwUmhRSEZKMFFLQmdRRCtOV0F4aUtSYmFpaFQ2OWlUCm5CbjlMNXU4eTZPQmRNK3J3dHh0blkzTlRnTHEvOVFGdEZ4ZDlFNTRwWGFha0Z3WE5Qb1VHSzJBd1MyOVVoblcKcHY1ekhiSW56S2lHUFpBdnNSaHdxbjRVTGQrZ1dId2QwTFdBblQ5U3RGNlpxRkIvV250ak02OEtpcTVwa29SdQo5amFoeFJqK1oyT3Z0Qzg1OFJXR1hraWdWUUtCZ1FEZ1huMHg0M2ZEc3J0UmI2ZHJqM2RYUkFWU3ZKTk8vdTN1CjhTN215REtRRnFCaEpWWXYyZC94MFNwZTdPMWtUZlEvSDdKVTVGbzFaL1hXSmh0MjAxR0U0UEUxZHVtU1VYMHYKanNrZm1ET0lISFZidG1jYnJMVGtLbThKUHNiQnhqL2FrdXJucFljUlprcjcvTE5wb2xLMWw0b0dSWERjd21tYQpuMFBQOVZlc21RS0JnR05ZWUx0Njl5QnlvRWdMeDV3SVNXL1ZCendGS1dVU1RZYUdwTGFCK0VUZjArd2JCclVrCkEyVUVjUlpkTTdpNzZnRjN4clBMNFNLTVhGKzhralI5RnJ2T1F5MmlvY20wallEVjZRTSszZjdzM3BRL2dHSDIKaXJnMHUrYUtidDRHRlU5d0FRQ0gxa1I4WmlVRUNaV0R5MTMxTlF0WFJISUtMTnFzQ1FsMkYxUHhBb0dBZWlwNAp1SVZHWE1RZ3drS3RRaCtudDFoT0hiS1g4dGRUN2xwY1UwU0dZM1l1ZFRiREt1TFMwUFEyd2VEM2tUbncxNzRHCmQ3cXVjNDNiaVhMT1dYRE9YaStzcHBQTXNlSzA2d1F2OW9nRTV3TzMwdXdLVUVORmEwYS9scVVieTc2S1hoVEkKMnNDQ1ByTVgrUVNyRzhWS2k1MnJoYWFod1ZJSjlpcnAzWDJVODJFQ2dZQnRCQjdzejc1SU1wbThqelduQmh3SAo5YVlRQTVqczd0NXRpT3I4OXNaR0xmUDg5dDdrR0RPajA1S1B0OCtrS1lEdXdFVUxZOGpzNnpUcmdvdStBQlEyCmNlSjFoVmZxWC8yTWYvb3F1eUEzS2dMemRLS0M2clFCY1hTVUR2SVR2SlFuWHV3eGsyQS9nZ09jNTZoRW1sNm4KdXdqN1JPQkRiUzNaVUwyZXFYb2s3QT09Ci0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K
```

Comment:
Should we add add a document to describe how to generate this cert so that we can regenerate it after it expires?

Reply:
I can add the comment, but those certs are valid for 100 years s oti should be enough